.github/workflows/chain-spec-snapshot-build.yml

```diff
@@ -14,7 +14,7 @@ on:
 jobs:
   chains-spec:
     runs-on: ${{ fromJson(github.repository_owner == 'autonomys' &&
-      '"runs-on=${{ github.run_id }}/runner=self-hosted-ubuntu-22.04-x86-64"' || '"ubuntu-22.04"') }}
+      '"runs-on=${{ github.run_id }}-${{ github.run_attempt }}/runner=self-hosted-ubuntu-22.04-x86-64/spot=false"' || '"ubuntu-22.04"') }}
```

Comment:
Including `spot=false` isn't that redundant since it's turned off in `runs-on.yml` file?